lib/menu.rb

```diff
@@ -0,0 +1,20 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    # I could have the menu read from a file in a private method
+    # at initialisation. I have chosen to keep things simple here for
+    # ease of testing
+    @dishes = { burrito: '6.80' }
```

Comment:
The customer doesn't have many choices here ;). Perhaps you could add more items to the menu so that the customer can add various dishes to the order.